packages/modules/src/modules/banners/designableBanner/components/CustomArticleCountCopy.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { from, headline, space } from '@guardian/source/foundations';
+
+const styles = {
+    container: css`
+        ${headline.xxxsmall({ fontWeight: 'bold' })}
+        font-size: 15px;
+        margin: 0 0 ${space[1]}px;
+
+        ${from.tablet} {
+            font-size: 17px;
+        }
+    `,
+};
+
+interface CustomArticleCountProps {
+    copy: string;
+    numArticles: number;
+}
+
+export function CustomArticleCountCopy({
```

Comment:
This component is used by the DesignableBanner, but was previously imported from the `WorldPressFreedomDay` banner, so I've moved it here